docker-container/auth.json

```diff
@@ -0,0 +1,8 @@
+{
+    "http-basic": {
+        "repo.magento.com": {
+            "username": "4a0f697f54f044079066cc6f96168815", 
+            "password": "c94815a6ac7e7d400a1e989bbe8c1be2"
```

Comment:
let's put this into env variable and put in 1pass

Reply:
I have removed this file. But I store the variables in the config, do you suggest a different location?